litmus-portal/frontend/src/theme/index.tsx

```diff
@@ -136,6 +150,13 @@ function customTheme(options: ThemeOptions) {
         passedTests: 'rgba(16, 155, 103, 0.2)',
         failedTests: 'rgba(202, 44, 44, 0.2)',
       },
+      projectDisplayColor: 'rgba(128, 128, 128)',
+      buttonEditProfile: 'rgba(0, 0, 0)',
+      homePageCardBackgroundColor: '#FFFFFF',
```

Comment:
can we have a common `card.background: #fff`

Reply:
No, we can not use the common `card.background` because as per design some card have different background color! Once we fix the design then we can remove this exceptional variable but a/c to current design these all are required.